specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/privateLinkService.json

```diff
@@ -740,9 +755,9 @@
           "$ref": "./virtualNetwork.json#/definitions/Subnet",
           "description": "The reference of the subnet resource."
         },
-        "publicIPAddress": {
-          "$ref": "./publicIpAddress.json#/definitions/PublicIPAddress",
-          "description": "The reference of the public IP resource."
+        "primary": {
+          "type": "boolean",
+          "description": "Whether the ip configuration is primary or not. (True/False)."
```

Comment:
I don't think you need to specify `(True/False)` in description since it's already a boolean property